content/en/docs/ambient/getting-started/setup/index.md

```diff
@@ -0,0 +1,127 @@
+---
+title: Install Ambient
+description: Download and Install Istio in Ambient mode.
+weight: 1
+owner: istio/wg-networking-maintainers
+test: yes
+---
+
+In this section you'll download Istio CLI (`istioctl`) and install Istio in ambient mode on your Kubernetes cluster. You'll need a Kubernetes cluster to proceed. If you don't have a cluster, you can use [kind](/docs/setup/platform-setup/kind) or any other [supported Kubernetes platform](/docs/setup/platform-setup).
+
+## 1. Download Istio CLI
+
+You'll use [Homebrew](https://brew.sh/) to download the Istio CLI:
+
+{{< text bash >}}
+$ brew install istioctl
+{{< /text >}}
+
+{{< quote >}}
+If you don't have Homebrew, follow these instructions to download and install the Istio CLI:
+
+{{< text syntax=bash snip_id=none >}}
+$ curl -L https://istio.io/downloadIstio | sh -
+$ cd istio-{{< istio_full_version >}}
+$ export PATH=$PWD/bin:$PATH
+{{< /text >}}
+
+{{< /quote >}}
+
+Verify the CLI is installed correctly by running the following command:
+
+{{< text syntax=bash snip_id=none >}}
+$ istioctl version
+no ready Istio pods in "istio-system"
+{{< istio_full_version >}}
+{{< /text >}}
+
+## 2. Install Istio onto your cluster
+
+Assuming your cluster is prepared, you can install Istio using the ambient mode profile. Run the following command:
+
+{{< text syntax=bash snip_id=install_ambient >}}
+$ istioctl install --set profile=ambient --skip-confirmation
+{{< /text >}}
+
+Note that it might take a minute for the Istio components to be installed. Once the installation completes you’ll get the following output that indicates all components have been installed successfully!
+
+{{< text syntax=plain snip_id=none >}}
+✔ Istio core installed
+✔ Istiod installed
+✔ CNI installed
+✔ Ztunnel installed
+✔ Installation complete
+{{< /text >}}
+
+You can verify the installed components using the following command:
+
+{{< text syntax=bash snip_id=verify_install >}}
+$ istioctl verify-install
+
```

Comment:
we can elide most of the items from this list?

Reply:
In my notes, I wrote "perhaps we need a --short option for the verify-install command" :) 

But yes, I'll remove this as the initial installation already outputs the checkmarks and verifies everything is installed.